.changeset/polite-melons-hide.md

```diff
@@ -0,0 +1,6 @@
+---
+"@commercetools-docs/gatsby-theme-api-docs": patch
+"@commercetools-docs/gatsby-theme-docs": patch
+---
+
+chore: initial e2e test setup for gatsby-theme-api-docs
```

Comment:
There should be no need to release a new version right? We're only adding e2e tests here.